lib/ansiblelint/__main__.py

```diff
@@ -117,6 +118,12 @@ def main() -> int:
     for match in sorted(set(matches)):
```

Comment:
@ssbarnea this is `sorted(set())` and the one below is just `sorted()`.
1) I think that both should be `sorted(set())`
2) Also, it'd be better to just safe this in a variable and reuse it in two places